### PR TITLE
Add a file for Blaine, MN

### PR DESCRIPTION
**Pull request by SavageSpaceLion**
The purpose of this pull request is to add a hometown file for Blaine, MN :soccer: 

Some :question: that I want you all to help me answer:
1. If you've been to Blaine, what did you do? What did you think of it?
1. If you have kids, what kinds of things do they like to do at night?
1. What kind of food do you like?

### CODE REVIEW

File: SavageSpaceLion-Blaine.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Blaine, MN
+
+### Restaurants
+- T-Box
+- Roadside
+- Bricks
+
+### Places the locals like
+- Lockness park
```
Comment (vpierson) [inline SavageSpaceLion-Blaine.md:9]: Is the lockness monster there? :fearful:

**Comment by vpierson** [comment]
1. I've never been to Blaine
1. I have cat children, and they like to sleep at night :cat: :zzz: 
1. I like :pizza:!

**Comment by SavageSpaceLion** [comment]
This pull request resolved #16

**Review by vpierson — APPROVED** [review]
:+1: :ship:

**Pull request merged**